label/label.go

```diff
@@ -0,0 +1,95 @@
+package label
+
+import (
+	"fmt"
+)
+
+// Test is a label associated with an individual test.
+// All test labels are known at compile time.
+type Test string
+
+const (
+	Timeout          Test = "timeout"
+	HeightTimeout    Test = "height_timeout"
+	TimestampTimeout Test = "timestamp_timeout"
+)
```

Comment:
Do we need a label for `relay packet`? (A better name may be `fungible transfer` or similar.)

Also, I now see there's a small hierarchy or set membership going on. I was at first confused by `Timeout` but now I see it encompasses all timeout tests. The `HeightTimeout` and other variants let you focus more.

Reply:
It crossed my mind to add an explicit description associated with each test label. If it wasn't immediately obvious now, it will be even less obvious as the label count grows. 

 I'll add that in the next iteration on this part of the code. Probably only for test labels because those are all determined at compile time.